csvcubed/tests/unit/utils/test_pandas_na_filter.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+from csvcubed.utils.pandas_na_filter import refiltered_pandas_read_csv
+
+csv_path = "/workspaces/csvcubed/csvcubed/tests/test-cases/utils/pandas_na_filter/code-list.csv"
```

Comment:
This URL won't work for developers who aren't using docker. Don't forget about the function you can use which finds the `test-cases` directory for you.